lib/python/file_classifier.py

```diff
@@ -38,6 +40,33 @@ def _error(cls, message):
         "Raise an exception with the given message."
         raise FileClassifierException, message
 
+
+    @classmethod
+    def _get_file_name_fields(cls, input_file, min_fields=6):
+        """Return the '_'-separated fields in the file name as a list.
+        Raise an exception if the number of fields is less than min_fields.
+        """
+        just_the_name = re.sub('\.nc$', '', os.path.basename(input_file))
```

Comment:
I don't know if you want to note it, but this function will work only on files ending with `nc`. And if it doesn't? Unexpected behaviour?